installation.md

```diff
@@ -40,7 +40,7 @@ $ dep self-update --upgrade
 You can install Deployer with composer:
 
 ``` sh
-$ composer require deployer/deployer:~3.0
+$ composer require deployer/deployer:~3.0 --dev
```

Comment:
I included it because you are not very likely to have your deployment library deployed in production, just like other dev libraries: phpunit, phpspec, cs fixer, etc.